README.md

```diff
@@ -303,6 +303,19 @@ ensures that the appropriate validation services are in place.
 _Note: This feature is still in development and we are looking for feedback on
 features and improvements in ways it can be used_
 
+### Database Setup
```

Comment:
I think this is confusing, because it is unclear that this section applies to running the console.  Is it harmful to just add `db:create db:migrate` to all the below examples, and not have it part of this section?  That will create the db if necessary, and not doing anything otherwise, right?

Reply:
Yup.